backend/metamodel/syson-sysml-metamodel/src/main/java/org/eclipse/syson/sysml/impl/MultiplicityRangeImpl.java

```diff
@@ -67,7 +71,11 @@ protected EClass eStaticClass() {
      */
     @Override
     public EList<Expression> getBound() {
-        List<Usage> data = new ArrayList<>();
+        List<Expression> data = new ArrayList<>();
+        data.add(getLowerBound());
+        if (!Objects.equals(getLowerBound(), getUpperBound())) {
```

Comment:
In EMF the equals method is not implemented. What are you tring to check?

Reply:
done